README.md

```diff
@@ -12,194 +12,129 @@ shared. These resources have a different lifecycle than resources in workspaces.
 
 ## Implementing a Landing Zone
 
-### Landing Zone Definition Factories and Landing Zone Definitions.
+Landing zone creation process is implemented as a [Stairway](https://github.com/DataBiosphere/stairway) flight represented by the following class `CreateLandingZoneFlight`. This flight consists of different steps one of which is step which 
+represents sub-flight to create Azure resources. This sub-flight is represented by the following class `CreateLandingZoneResourcesFlight`. This class utilizes specific steps' provider to define the list of steps required to create Azure resources.
 
-Landing zones are implemented using the factory pattern; the factory creates *Landing Zone Definitions* (LZDs).
+Below are lists of changes required to introduce new landing zone:
+1) Implement specific steps which are responsible for Azure resource creation (see example below for `CreateVnetStep`)
+2) Implement specific steps' definition provider (by implementing `StepsDefinitionProvider` interface)
+3) Introduce new landing zone type by extending `StepsDefinitionFactoryType`
+4) Update mapping at `LandingZoneStepsDefinitionProviderFactory` based on landing zone type from section 3. This mapping is used by sub-flight to get flights' steps to create specific Azure resources.
```

Comment:
Suggested: Update the mapping at `LandingZoneStepsDefinitionProviderFactory` based on new landing zone type. This mapping is used by the sub-flight `CreateLandingZoneResourcesFlight` to get the steps to create specific Azure resources.